src/content/lib/prefs.js

```diff
@@ -1,13 +1,23 @@
 // const Store = window.require('electron-store');
-import prefDefaults from "../../config/pref_defaults";
+const prefDefaults = {
+  // The user's bugzilla email
+  bugzilla_email: "",
+  // Disable all client-side network requests
+  offline_debug: false,
+  // Disable the user cache for bug views
+  disable_cache: false,
+};
 
 class Store {
-  get(key) {
-    return localStorage.getItem(key);
+  get(key, defaultValue) {
+    if (localStorage.getItem(key) === null && key in prefDefaults) {
+      this.set(key, defaultValue);
+    }
+    return JSON.parse(localStorage.getItem(key));
```

Comment:
btw not sure why we need JSON here. all the prefs are already valid JSON types

Reply:
localStorage stores values as strings. For `offline_debug` and `disable_cache`, we want a boolean `false`, not a string `"false"` (which happens without this JSON.parse)

Reply:
Makes sense, we just don't need JSON.stringify

Reply:
well I guess if user plugs in `"true"` to bugzilla_email it could bork something, so go ahead and keep it